server/views/pages/contacts/manage/contactConfirmation/addresses.njk

```diff
@@ -0,0 +1,71 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{{ govukSummaryList({
+    card: {
+        title: {
+            text: "Addresses"
+        },
+        actions: {
+            items: [
+              {
+                href: "#",
+                text: "View all addresses"
+              }
+            ]
+        }
+    },
+    rows: [{
+        key: {
+            text: "Address"
+        },
+        value: {
+            html: '',
+            classes: 'confirm-address-value'
```

Comment:
I would give all these things a data-qa for selection instead of classes, unless they actually need styling?
 `attributes: {"data-qa": "confirm-address-value"},`

Reply:
data-qa attribute added to replace classes